src/points/cardSort.ts

```diff
@@ -0,0 +1,18 @@
+import { Props as ActivityCardProps } from 'src/points/ActivityCard'
```

Comment:
all good here but it made me think that perhaps we should return an order from the getPointsConfig endpoint instead in the future...

Reply:
oh, it would be great to get the list already properly sorted from the config!

one challenge I'm thinking about is how to sort activities alphabetically in different languages. alphabetical sort is needed when we have multiple activities of the same points value. 

or we can ask the marketing team to define the order 😅